### PR TITLE
Feat/tapc 1731 instance new page v1

### PR DESCRIPTION
**Pull request by fredericvilcot**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/pci-instances`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #1731 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

This PR:

- declares the `/new` route 
- initializes the new `CreateInstance` component
- enables links in `Create an instance` buttons in `Instances` and `Onboarding` pages
- refactors the way pages are nested under `instances` folder
- improves `Breadcrumb` component

### CODE REVIEW

**Review by Tsiorifamonjena — APPROVED** [review]

File: packages/manager/apps/pci-instances/src/pages/instances/create/CreateInstance.page.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import { OsdsDivider } from '@ovhcloud/ods-components/react';
+import {
+  Breadcrumb,
+  TBreadcrumbProps,
+} from '@/components/breadcrumb/Breadcrumb.component';
+import { useHidePreloader } from '@/hooks/hidePreloader/useHidePreloader';
+
+const CreateInstance: FC = () => {
+  const project = useRouteLoaderData('root') as TProject;
+  const { t } = useTranslation(['create', 'common']);
```
Comment (ppprevost) [inline packages/manager/apps/pci-instances/src/pages/instances/create/CreateInstance.page.tsx:19]: don't need to import if you do : common:create_instance

**Review by ppprevost — CHANGES_REQUESTED** [review]

File: packages/manager/apps/pci-instances/src/pages/instances/Instances.page.tsx [hunk 2]
```diff
@@ -309,6 +313,7 @@ const Instances: FC = () => {
             variant={ODS_BUTTON_VARIANT.stroked}
             color={ODS_THEME_COLOR_INTENT.primary}
             inline
+            onClick={createInstance}
```
Comment (SimonChaumet) [inline packages/manager/apps/pci-instances/src/pages/instances/Instances.page.tsx:316]: You should use href attribute instead

File: packages/manager/apps/pci-instances/src/pages/instances/onboarding/Onboarding.page.tsx [hunk 3]
```diff
@@ -39,17 +45,26 @@ const Onboarding: FC = () => {
...
+    <Navigate to={rootUrl} />
   ) : (
     <PageLayout>
-      {project && <Breadcrumb projectLabel={project.description} />}
+      {project && <Breadcrumb projectLabel={project.description ?? ''} />}
       <OnboardingLayout
         title={t('common:instances_title')}
         img={{ src: InstanceImageSrc }}
         orderButtonLabel={t('common:create_instance')}
+        onOrderButtonClick={createInstance}
```
Comment (SimonChaumet) [inline packages/manager/apps/pci-instances/src/pages/instances/onboarding/Onboarding.page.tsx:67]: This should also be a link

File: packages/manager/apps/pci-instances/src/pages/instances/create/CreateInstance.page.tsx (hunk 1 above)
Comment (SimonChaumet) [reply packages/manager/apps/pci-instances/src/pages/instances/create/CreateInstance.page.tsx:19]: To me it is better to import it, it is the way in the documentation

**Review by ppprevost — APPROVED** [review]

**Review by SimonChaumet — APPROVED** [review]

**Pull request merged**